markdown/events/2022/bytesize-survey_results.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Bytesize: community survey results'
+subtitle: Phil Ewels - Sequera labs
+type: talk
+start_date: '2022-05-24'
+start_time: '13:00 CET'
+end_date: '2022-05-24'
+end_time: '13:30 CET'
+location_url:
+  - https://kth-se.zoom.us/j/68390542812
+---
+
+# nf-core/bytesize
+
+Join us for our **weekly series** of short talks: **“nf-core/bytesize”**.
+
+Just **15 minutes** + questions, we will be focussing on topics about using and developing nf-core pipelines.
+These will be recorded and made available at <https://nf-co.re>
+It is our hope that these talks / videos will build an archive of training material that can complement our documentation. Got an idea for a talk? Let us know on the [`#bytesize`](https://nfcore.slack.com/channels/bytesize) Slack channel!
+
+## Bytesize: nf-core/community survey results
+
+This week, Phil Ewels ([@ewels](https://github.com/ewels/)) will give an overview over the results of the nf-core community survey.
```

Comment:
```suggestion
## Bytesize: The Nextflow and nf-core community survey

At the start of 2022, nf-core joined the Nextflow community to do a joint survey entitled _"The State of the Workflow 2022:  Nextflow and nf-core community survey"_ (see the [announcement blog post](https://seqera.io/blog/the-state-of-the-workflow-the-2022-nextflow-and-nf-core-community-survey/)). This follows on from the [2021 edition](https://seqera.io/blog/state-of-nextflow-2021-results/) but with some additional questions, including some concerning nf-core community strength and engagement.

This week, Phil Ewels ([@ewels](https://github.com/ewels/)) will give an overview over the key results of the survey. If that's not enough excitement for you, Evan Floden ([@evanfloden](https://github.com/evanfloden) will wrap up with a live-draw to select the winners of the survey draw: one Apple Macbook Air M1 up for grabs alongiside ten sustainability-certified Seqera Labs merchandise packs.. 🤩 💚 💻 👕 
```

## Bytesize: The Nextflow and nf-core community survey

At the start of 2022, nf-core joined the Nextflow community to do a joint survey entitled _"The State of the Workflow 2022:  Nextflow and nf-core community survey"_ (see the [announcement blog post](https://seqera.io/blog/the-state-of-the-workflow-the-2022-nextflow-and-nf-core-community-survey/)). This follows on from the [2021 edition](https://seqera.io/blog/state-of-nextflow-2021-results/) but with some additional questions, including some concerning nf-core community strength and engagement.

This week, Phil Ewels ([@ewels](https://github.com/ewels/)) will give an overview over the key results of the survey. If that's not enough excitement for you, Evan Floden ([@evanfloden](https://github.com/evanfloden) will wrap up with a live-draw to select the winners of the survey draw: one Apple Macbook Air M1 up for grabs alongiside ten sustainability-certified Seqera Labs merchandise packs.. 🤩 💚 💻 👕